<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image</string>
    </property>
   </widget>
   <!-- 图像显示区域 -->
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Display</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 功能按钮 -->
   <widget class="QPushButton" name="button_add_point">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Endpoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_measure_distance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure Distance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_measure_angle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure Angle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_measure_ratio">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure Ratio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_logs">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Logs</string>
    </property>
   </widget>
   <!-- 日志显示区域 -->
   <widget class="QTextEdit" name="textEdit_logs">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>700</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
